python/ray/tests/test_actor_failures.py

```diff
@@ -11,9 +11,9 @@
 import ray.ray_constants as ray_constants
 import ray.test_utils
 import ray.cluster_utils
-from ray.test_utils import (relevant_errors, wait_for_condition,
-                            wait_for_errors, wait_for_pid_to_exit,
-                            generate_internal_config_map)
+from ray.test_utils import (
+    relevant_errors, wait_for_condition, wait_for_errors, wait_for_pid_to_exit,
+    generate_internal_config_map, get_non_head_nodes, get_other_nodes)
```

Comment:
add a trailing "," and put one item per line.

Reply:
> add a trailing "," and put one item per line.

It's auto-lint by yapf.